src/template.njk

```diff
@@ -0,0 +1,59 @@
+{% from "./components/skip-link/macro.njk" import govukSkipLink %}
+{% from "./components/header/macro.njk" import govukHeader %}
+{% from "./components/footer/macro.njk" import govukFooter %}
+
+{% block pageStart %}{% endblock %}
+<!DOCTYPE html>
+<html lang="{{ htmlLang | default('en') }}" class="govuk-template">
```

Comment:
how does one extend this to add custom classes or ie specific classes?

Reply:
Makes sense, I'll add that feature